docs/optimization.md

```diff
@@ -259,7 +259,9 @@ optimization { execution_accelerators {
 
 As Triton starts you should check the console output and wait until
 Triton prints the "Staring endpoints" message. ONNX model loading can
-be significantly slower when TensorRT optimization is enabled.  Now
+be significantly slower when TensorRT optimization is enabled. In production you can use [model
+warmup](model_configuration.md#model-warmup) to avoid this model
+startup/optimization slowdown. Now
```

Comment:
Fix formatting in this line. Otherwise LGTM
Thanks for your contribution @rmporsch. If this is our first PR, could you complete the [CLA ](https://github.com/NVIDIA/triton-inference-server/blob/master/CONTRIBUTING.md)and send it out to triton-cla@nvidia.com, if you have not already done so before?